README.md

```diff
@@ -38,7 +59,8 @@ Application secrets are not included in the code repository. Below are the instr
 
 ###### DEV
 
-Request for **developer-local-properties.yaml** file from Admin, which should be copied to below location
+Create **developer-local-properties.yaml** file under root of `resource` folder and set the application.yaml
+placeholders.
 

```

Comment:
Didn't understand this wording under PROD:
"Secrets will be set as an environment variables on the deployment environment.". What does it mean?